_Courses/programming-with-text/11-chrome-extensions/11.8-cross-browser-extensions.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Cross-browser Extensions"
+video_number: 78
+date: 2017-11-28
+video_id: RJtjp-Vtwjs
+repository: /A2Z/session11/11-8_Cross-Browser_Extensions
```

Comment:
This links to nowhere. It should be `/A2Z/session11/11-08_Cross-Browser_Extensions`